### ISSUE TITLE
Induction principle name of a set-declared type not parsed correctly

### ISSUE THREAD

**Issue by fblanqui** [Deducteam/lambdapi#489]
```
constant symbol Prop : TYPE
set declared "π"
injective symbol π : Prop → TYPE

set builtin "Prop"     ≔ Prop
set builtin "P"     ≔ π

set declared "𝔹"
inductive 𝔹 : TYPE ≔
 | true  : 𝔹
 | false : 𝔹
type ind_𝔹

16:47 ~/src/lambdapi (master) lp tmp/ind_bool_unicode.lp 
[/home/blanqui/src/lambdapi/tmp/ind_bool_unicode.lp, 12:5-9] Unbound symbol or variable [ind_].
```

**Comment by amelieled** [comment]
This issue will be fixed with the new parser.

**Comment by fblanqui** [comment]
Fixed in #516 .
